crates/router-controller/src/controller.rs

```diff
@@ -127,8 +131,31 @@ impl Controller {
             0 => None,
             1 => Some(&candidates[0].url),
```

Comment:
Result from here should be also registered in counters.

Reply:
👍 
but registration in `request_tracker` seems unnecessary to me so i'll just increase worker rate